lib/jmap/mail/mailbox/get_mailbox_method.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:jmap_dart_client/http/converter/account_id_converter.dart';
+import 'package:jmap_dart_client/http/converter/id_converter.dart';
+import 'package:jmap_dart_client/http/converter/properties_converter.dart';
+import 'package:jmap_dart_client/jmap/account_id.dart';
+import 'package:jmap_dart_client/jmap/core/capability/capability.dart';
+import 'package:jmap_dart_client/jmap/core/method/request/get_method.dart';
+import 'package:jmap_dart_client/jmap/core/request/request_invocation.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'get_mailbox_method.g.dart';
+
+@IdConverter()
+@AccountIdConverter()
+@PropertiesConverter()
+@JsonSerializable()
+class GetMailboxMethod extends GetMethod {
+  GetMailboxMethod(AccountId accountId) : super(accountId);
+
+  @override
+  MethodName get methodName => MethodName('Mailbox/get');
+
+  @override
+  Set<CapabilityIdentifier> get requiredCapabilities => {
```

Comment:
requiredCapabilities should not be part of the output json right?

Reply:
yes, it not belongs to `arguments` but it will be put in `invocation`